packages/vscode-extension/src/commands/ManageAccounts.ts

```diff
@@ -57,6 +60,8 @@ export default class ManageAccounts extends BaseAction {
 	}
 
 	protected async execute() {
+		this.vsConsoleLogger.addExecutionDetailsToLog();
```

Comment:
Is this one really needed? I think you have already "initialized" the logger to show the execution details at line 52.

Reply:
It looks like in the vscode-extension, the commands are being cached. So once ManageAccounts is initialized, it will not be initialized again on the subsequent calls to the command. If I don't add this line per execute, then the execution details will only be shown on the first call to the ManageAccount command.

Reply:
Oh! Thanks for the explanation. Maybe we should add a comment to remember in the future?

Reply:
Yeah, I agree. I added the comment.

Reply:
Great!